next-cloudinary/src/components/CldUploadWidget/CldUploadWidget.tsx

```diff
@@ -264,6 +267,13 @@ const CldUploadWidget = ({
             ...instanceMethods
           });
         }
+
+        const widgetEventAction = `${widgetEvent}Action` as keyof typeof props;
+
+        if ( widgetEventAction && typeof props[widgetEventAction] === 'function' ) {
+          const action = props[widgetEventAction] as CldUploadEventAction;
+          uploadResult.event === 'success' && action(uploadResult);
```

Comment:
i think we can remove the 'success' check here right? as long as the function exists and its under the watched events, which at this point i believe both should be true, we should be good, right?